### PR TITLE
Presets: Two specific tree presets

### PR DESCRIPTION
**Pull request by tordans**
Most trees in Europe tend to fall into one of the two categories
- Broadleaved Tree (deciduous)
- Needleleaved Tree (evergreen)

There are cases, where this is not true, of course, but in my experience the majority fits into those two groups.

Those two specific presets would make it a lot easier to add a tree of those groups and apply all needed tags in one go. Variations are also easier to change, eg. when only `leaf_cycle` needs to change to `semi_evergreen`.

- https://wiki.openstreetmap.org/wiki/Tag:natural%3Dtree#How_to_map
- https://wiki.openstreetmap.org/wiki/Key:leaf_cycle
- https://wiki.openstreetmap.org/wiki/Key:leaf_type

---

My overall goal is, to be able to type "laubbaum" in GoMap and have all tree tags filled out…

---

Open questions:

- [x] How does the search priority work => OK https://github.com/openstreetmap/id-tagging-schema/pull/956#issuecomment-1633559615
- [x] Do the icons look good => OK https://github.com/openstreetmap/id-tagging-schema/pull/956#issuecomment-1633560808

### CODE REVIEW

**Comment by tordans** [comment]
Update: I need to change the icon… will look into it later.

**Comment by tordans** [comment]
Update: I change to use `maki-park` for broadleaved trees since that icon looks better then roetgen and it is more consistent.

image Bildschirm­foto 2023-07-12 um 22 29 54

**Comment by tordans** [comment]
New question @tyrasd : How do I get the preset to render the `leave_cycle` and `leaf_type` field for the new presets even though those are part of the `tags` object?

Right now it looks like this ([Link](https://pr-956--ideditor-presets-preview.netlify.app/id/dist/#background=Bing&disable_features=boundaries&map=22.00/52.47571/13.44494))… which makes it hard to change the given settings for `leave_cycle` and `leaf_type`.

Bildschirm­foto 2023-07-12 um 22 30 17

**Comment by tyrasd** [comment]
just add the fields again to the individual presets: they are only omitted in the inherited/referenced fields from other presets (and also present as `tags` of the preset), but kept if directly included in the `fields` list.

**Comment by tordans** [comment]
> * [ ]  How does the search priority work

I think that is fine as is…

**Comment by tordans** [comment]
> - [x] Do the icons look good

I think this looks good now…

image

**Comment by tordans** [comment]
@tyrasd thanks. This is ready for review, now :)

PS: I created two PRs in the schema repo to improve the docs. https://github.com/ideditor/schema-builder/pulls/tordans

**Comment by tyrasd** [comment]
I've added (for now unsearchable) presets for the not-so-common-but-still-possible combinations (needleleaved+deciduous & broadleaved+evergreen) and the leaf-type-only versions. This should make the mapping more seamless with existing data and will show the correct icon for all leaf_type-mapped trees. Also, I've changed the icon for the generic Tree preset, such that it is visually different in the presets list and on the map.

What do you think?

File: data/presets/natural/tree/_needleleaved.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        "natural": "tree",
+        "leaf_type": "needleleaved"
+    },
+    "terms": [
+        "conifer",
+        "needleleaved",
+        "softwood"
+    ],
+    "name": "Needleleaved Tree"
+}
```
Comment (tordans) [inline data/presets/natural/tree/_needleleaved.json:22]:
```suggestion
    "name": "Needleleaved Tree",
    "searchable": false
}
```

File: data/presets/natural/tree/needleleaved/_deciduous.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        "leaf_type": "needleleaved",
+        "leaf_cycle": "deciduous"
+    },
+    "terms": [
+        "conifer",
+        "deciduous",
+        "needleleaved",
+        "softwood"
+    ],
+    "name": "Needleleaved Tree (deciduous)"
```
Comment (tordans) [inline data/presets/natural/tree/needleleaved/_deciduous.json:23]:
```suggestion
    "name": "Needleleaved Tree (deciduous)",
    "searchable": false
```

**Review by tordans — COMMENTED** [review]
Thanks for the update @tyrasd.

I was holding back on adding more detailed presents to not overwhelm the translators. But given how many trees we have in OSM, adding those presents makes it a lot clearer, so that is good!

I still do not like the roentgen-tree icon inside the map in. It looks like an ice cream cone to me. However, given having the general preset different from the more precise presets is good idea. Ideally we would have a "left leaves, right needle" icon in temaki.

I saw that a few of the unsearchable presets did not have the unsearchable flat. I will add this after submitting this comment.

    "searchable": false

**Pull request merged**